bugsnag-android-core/src/main/java/com/bugsnag/android/Stacktrace.kt

```diff
@@ -6,7 +6,7 @@ import java.io.IOException
  * Serialize an exception stacktrace and mark frames as "in-project"
  * where appropriate.
  */
-internal class Stacktrace : JsonStream.Streamable {
+class Stacktrace : JsonStream.Streamable {
```

Comment:
This should remain `internal` - the default visibility in Kotlin is public, so removing this modifier would expose `Stacktrace` as part of the public API.

Reply:
Hmm I had to expose it in order for the JNI plugin to see it. Is there a way to keep it internal but allow the JNI plugin visibility?

Reply:
Ah, I see. Kotlin decompiles down to Java bytecode and [mangles names](https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#visibility) for methods in internal classes I believe. You could attempt using [`@JvmName`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-name/) or even making the method private to avoid name mangling, which is what the `AnrPlugin` does: https://github.com/bugsnag/bugsnag-android/blob/69aba9a05e5de94101a03ebb24bf49d7d6438085/bugsnag-plugin-android-anr/src/main/jni/anr_handler.c#L43